### PR TITLE
Fix problem with rocksdb on alpine based image

### PR DESCRIPTION
**Pull request by iemejia**
Alpine has a libc6-compat library that makes rocksdb work on it

### CODE REVIEW

**Comment by iemejia** [comment]
@patricklucas also worth taking a look, and also if your tests go well worth releasing finally the alpine version image.

**Comment by patricklucas** [comment]
Hmm, as noted in #14 I tried the libc6-compat library but still got a (different) error. That doesn't happen now when you use the same 'steps to reproduce'?

**Comment by iemejia** [comment]
I tested exactly the same steps you mention with no exception so far, maybe it is also because the image is now based on a more recent version of alpine I think.

**Comment by iemejia** [comment]
@patricklucas this has been waiting for some time to be merged, PTAL.

File: Dockerfile-alpine.template [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
 FROM openjdk:8-jre-alpine
 
-# Install dependencies, Bash, and su-exec for easy step-down from root
-RUN apk add --no-cache bash snappy 'su-exec>=0.2'
+# Install dependencies, bash and su-exec for easy step-down from root
```
Comment (patricklucas) [inline Dockerfile-alpine.template]: This change seems unnecessary, but not a blocker to merging I suppose.

**Comment by patricklucas** [comment]
Haven't had a chance to test it, but I'll take your word for it.

**Comment by patricklucas** [comment]
I caved and decided to properly verify this—indeed, if you use an older version of alpine even with libc6-compat, the crash occurs, whereas in the version of alpine that the current version of openjdk:8-jre-alpine builds on, it does not.

I'll push the new Dockerfiles upstream shortly.

**Comment by honestAnt** [comment]
add this in dockerfile;

**Pull request merged**